time data
geometry_msgs/Transform cam2obj
geometry_msgs/Transform base2tool0
float64 time_cur_sec_msg
float64 time_pub_sec_msg
